### ISSUE TITLE
Add a CLI executable for interacting with WT (`wtcli.exe` or something)

### ISSUE THREAD

**Issue by 3N4N** [microsoft/terminal#11616]
# Description of the new feature/enhancement

With the command `sendInput` I can invoke shell commands with a key-binding. But that would send the input directly to the pane currently in focus. I want to invoke a **shell** command in the background. Kind of like how `moveFocus` command operates.

The specific use-case I want this for is navigating between vim and wt splits with one set of keybindings. As of now, I have to use two sets: one for vim (`ctrl-h/j/k/l`) and another for wt (`ctrl-alt-h/j/k/l`). I'm used to tmux and had a config which achieved exactly this purpose.

I can do half of the work from vim: when inside vim, I can check if there are other vim splits in the direction I want to move focus toward, and if not, I move over to the wt split in that direction, with the command `wt -w 0 mf <direction>`.

But that's only half the work. Now I have to configure wt to send the key to vim if the current wt split is harboring a vim spawn. In tmux, I had a way to --

1. See if the focused split has vim open
    ```tmux
    tmux display-message -p '#{pane_current_command}' | grep -iq vim
    ```
2. Bind a key to a complex bash command
    ```tmux
    bind -n C-h run \
             "((tmux list-panes -F '#F' | grep -q Z && tmux send-keys C-h) || \
             ((tmux display-message -p '#{pane_current_command}' | grep -iq vim \
               && tmux send-keys C-h) || tmux select-pane -L ))"
    ```

None of which is achievable in wt. Or I haven't found a way.

By the way, I noticed that the shell command `wt -w 0 mf <direction>` is slower than the native `moveFocus` command. And I think the shell variant even opens a new wt window for a split second. Why is that?

**Comment by zadjii-msft** [comment]
Hey sorry for leaving this untriaged for so long. 

This thread's a little hard to parse, but I suppose ultimately the request here was for something like 

```
wt display-message -p '#{pane_current_command}'
```

(and other commandline output about the state of `wt`)

That's kinda **H**ard at the moment. This comment has details: https://github.com/microsoft/terminal/issues/4570#issuecomment-586551445

We actually don't have an issue open tracking a separate CLI exe for interacting with `wt`. Coulda swore we did, but until we do, this thread is gonna become that one. Thanks!

**Comment by 3N4N** [comment]
@zadjii-msft 

> Hey sorry for leaving this untriaged for so long.
> 
> This thread's a little hard to parse, but I suppose ultimately the request here was for something like
> 
> ```
> wt display-message -p '#{pane_current_command}'
> ```
> 
> (and other commandline output about the state of `wt`)
> 
> That's kinda **H**ard at the moment. This comment has details: [#4570 (comment)](https://github.com/microsoft/terminal/issues/4570#issuecomment-586551445)
> 
> We actually don't have an issue open tracking a separate CLI exe for interacting with `wt`. Coulda swore we did, but until we do, this thread is gonna become that one. Thanks!

Hi, I understand the issues (kind of). But even after two years I sorely miss Vim-Tmux-Navigator.  So I decided to see if I can tweak the source code of Terminal myself.

I found the part where "moveFocus" action is handled (see below). Can I do something like the following (please 🥺) ?

```
std::shared_ptr<Pane> Pane::NavigateDirection(const std::shared_ptr<Pane> sourcePane, const FocusDirection& direction, const std::vector<uint32_t>& mruPanes)
{
  string tabTile = getTabTile();
  if (tabtitle.startsWith("Vim")) {
    sendKeyToTerminal(convertDirectionToPressedKey(direction));
    return nullptr;
  }
  // rest of the code
}
```
